### ISSUE TITLE
displaying output

### ISSUE THREAD

**Issue by rhalder2023** [ultralytics/ultralytics#15626]
### Search before asking

- [X] I have searched the Ultralytics YOLO [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hey, so if my show_labels and show_boxes are False in the predict function and 
              preds = model.predict(image, verbose = True, agnostic_nms = True, show_conf=False, show=False)

                plotted = preds[0].plot()

                cv.imshow("predicted",plotted)
                cv.waitKey(0)
                cv.destroyAllWindows()
                
   this is how im plotting, but these images show everything all the things i dont want
   
   and your show=True, just keeps popping up images because im running this on a list of images, and there are a lot of images.
   
   how do i turn these things off without having to use your show argument

**Comment by rhalder2023** [comment]
I also tried the show_labels=False, it still showed everything

**Comment by Skillnoob** [comment]
You need to put those arguments into `plot()`

**Comment by Breezewrf** [comment]
Maybe you can try `plotted = preds[0].plot(show=False, labels=False, probs=False)` 
Find more details on `plot()` https://github.com/ultralytics/ultralytics/tree/main/ultralytics/engine/results.py

**Comment by glenn-jocher** [comment]
You can disable unwanted elements by passing the appropriate arguments directly to the `plot()` method like this: `plotted = preds[0].plot(show=False, labels=False, probs=False)`. For more details, refer to the `plot()` method documentation.

**Comment by rhalder2023** [comment]
> You can disable unwanted elements by passing the appropriate arguments directly to the `plot()` method like this: `plotted = preds[0].plot(show=False, labels=False, probs=False)`. For more details, refer to the `plot()` method documentation.

Oh yeah it totally worked, i was using the same arguments for predict in plot that's why it was not working. But thanks for that. Its working.

**Comment by rhalder2023** [comment]
Btw i had one more question regarding something else. Do i make a new issue? 
If i dont need to make a new issue, i wanted to ask about model training,

Like say im using the yolo8m-seg model, and its detecting objects pretty well but not classifying them as well, so is it possible to freeze the weights in the detection layers and just finetune the weights in the classification head through your api?

**Comment by Skillnoob** [comment]
yes, there is the `freeze` parameter @rhalder2023

**Comment by rhalder2023** [comment]
> freeze

Im not sure where to use that, so have to go over the api docs for it

**Comment by glenn-jocher** [comment]
You can use the `freeze` parameter in the `train` method to specify the number of layers to freeze. For example, `model.train(data='data.yaml', epochs=10, freeze=10)` will freeze the first 10 layers. Check the API documentation for more details.

**Comment by rhalder2023** [comment]
> You can use the `freeze` parameter in the `train` method to specify the number of layers to freeze. For example, `model.train(data='data.yaml', epochs=10, freeze=10)` will freeze the first 10 layers. Check the API documentation for more details.

Ah thats exactly what i was looking for. Thank Youu

**Comment by glenn-jocher** [comment]
Glad to hear that helped! If you have any more questions, feel free to ask.
